pkgs/development/compilers/llvm/16/libcxxabi/default.nix

```
@@ -68,6 +68,12 @@ stdenv.mkDerivation rec {
   ] ++ lib.optionals (stdenv.hostPlatform.useLLVM or false) [
     "-DLLVM_ENABLE_LIBCXX=ON"
     "-DLIBCXXABI_USE_LLVM_UNWINDER=ON"
+  ] ++ lib.optionals ((stdenv.hostPlatform.useLLVM or false) || stdenv.hostPlatform.isDarwin) [
```

I opened #247066 to fix pkgsStatic on Darwin with LLVM 16. `enableShared` needs to be added to the condition, or the static libc++abi will fail to build.

```suggestion
  ] ++ lib.optionals ((stdenv.hostPlatform.useLLVM or false) || (stdenv.hostPlatform.isDarwin && enableShared) [
```

---

Done